local.properties

```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+#Mon Sep 24 14:24:11 SGT 2018
+sdk.dir=C\:\\Program Files (x86)\\Android\\android-sdk
```

Comment:
this file should be in .gitignore, as it contains info specific to ur local config